include/deal.II/lac/constraint_matrix.h

```diff
@@ -32,8 +32,6 @@
 #include <utility>
 #include <complex>
 
-#include <boost/scoped_ptr.hpp>
-
```

Comment:
If you find places such as this, feel free to submit the patch separately -- it's logically independent of everything else.